libs/brig-types/src/Brig/Types/User.hs

```diff
@@ -527,6 +534,13 @@ instance FromJSON ManagedByUpdate where
 instance ToJSON ManagedByUpdate where
     toJSON m = object ["managed_by" .= mbuManagedBy m]
 
+instance FromJSON RichInfoUpdate where
+    parseJSON = withObject "rich-info-update" $ \o ->
```

Comment:
The description should be either the Haskell type or some English phrase; the dashes are odd.

Reply:
All other update types (that existed before me) use hyphens. I'm not sure why.